### ISSUE TITLE
Failed to execute process ''.

### ISSUE THREAD

**Issue by dalizard** [JeanMertz/chruby-fish#8]
This is OSX Mavericks, running fish shell installed via `brew install https://raw.githubusercontent.com/JeanMertz/chruby-fish/master/homebrew/chruby-fish.rb`. Inside the `~/.config/fish/config.fish` I have:

```
source /usr/local/share/chruby/chruby.fish
source /usr/local/share/chruby/auto.fish
```

After loading a new session, I get the following error:

```
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
Failed to execute process ''. Reason:
The file '' does not exist or could not be executed.
set: Warning: path component /Users/dimitar/.rubies/2.1.1/lib/ruby/gems/2.1.0/bin may not be valid in PATH.
set: No such file or directory
~ $
```

The PATH error is present, because the bin folder does not exist. However, what is the reason behind the first error? If I remove the `source /usr/local/share/chruby/auto.fish
` line, then the first error is gone. Any suggestions?

**Comment by memoht** [comment]
Also so happy to have found this thread. I use ruby-install and giving chruby-fish a spin tonight. I don't have a /bin directory under `.rubies/ruby-2.2.0/lib/ruby/gems/2.2.0/` folder either. Thankful you created this project.

**Comment by JoshCheek** [comment]
I had this issue and others. I deleted the `-l` from the [bash invocation](https://github.com/JeanMertz/chruby-fish/blob/01e7d18a367594ae3eabcc2532e776041f650d8a/share/chruby/chruby.fish#L54) and everything started working fine. 

I understand that means I told it to launch bash as a "non-login" shell. After a lot of Googling, I found out that a "login-shell" is a shell for when you first log into a computer. Not really clear to me why that matters, I think maybe it's supposed to set up env vars for the first time, and then future bash's are expected to be subprocesses?

In my case, I had stuff in that file for setting keybindings, which were blowing up (something about not being in line-editing mode, probably from the `-c`) Maybe that means those commands should go in the `~/.bashrc` instead, but honestly, I looked for like an hour, and read the INVOCATION section of the man page like 3 times, and I can't find a straightforward explanation of what the different files are for and when they are used.

So, I have no idea why the `-l` flag was added (in [this](https://github.com/JeanMertz/chruby-fish/commit/52bb9f4e15787387dd5ae28da551e186dfa68cd7?diff=unified) commit), or what it really means, but when I deleted it, this problem and several others stopped.
